src/main/java/com/vinylteam/vinyl/util/PriceUtils.java

```diff
@@ -0,0 +1,42 @@
+package com.vinylteam.vinyl.util;
+
+import com.vinylteam.vinyl.entity.Currency;
+import lombok.extern.slf4j.Slf4j;
+import org.jsoup.nodes.Document;
+
+import java.util.Optional;
+
+@Slf4j
+public class PriceUtils {
```

Comment:
This class methods don't fit for other parsers. You need to indicate that this one is for juno parser

Reply:
Not only for juno parser, it is used in clone.nl parser too. It was the main idea to extract common methods for price string parsing